python/cugraph/cugraph/community/louvain.py

```diff
@@ -41,17 +42,24 @@ def louvain(G, max_iter=100, resolution=1.0):
         The current implementation only supports undirected graphs.
 
     max_iter : integer, optional (default=100)
-        This controls the maximum number of levels/iterations of the Louvain
+        This controls the maximum number of levels of the Louvain
         algorithm. When specified the algorithm will terminate after no more
-        than the specified number of iterations. No error occurs when the
+        than the specified number of levels. No error occurs when the
         algorithm terminates early in this manner.
 
-    resolution: float/double, optional (default=1.0)
+    resolution: float, optional (default=1.0)
```

Comment:
Dropped double from the documentation.  Couldn't find a way to properly support both float and double through the cython code.

Reply:
Can't see a reason why a double would be necessary.